tests/draft6/optional/format.json

```diff
@@ -47,6 +52,55 @@
         ]
     },
     {
+        "description": "validation of URI References",
+        "schema": {"format": "uri-reference"},
+        "tests": [
+            {
+                "description": "a valid URI",
+                "data": "http://foo.bar/?baz=qux#quux",
+                "valid": true
+            },
+            {
+                "description": "a valid protocol-relative URI Reference",
+                "data": "//foo.bar/?baz=qux#quux",
+                "valid": true
+            },
+            {
+                "description": "a valid relative URI Reference",
+                "data": "/abc",
+                "valid": true
+            },
+            {
+                "description": "an invalid URI Reference",
+                "data": "\\\\WINDOWS\\fileshare",
+                "valid": false
+            },
+            {
+                "description": "a valid URI Reference",
+                "data": "abc",
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "format: uri-template",
```

Comment:
I see. Actually the spec only defines interpolation ({...}), everything outside curly braces can be anything, it doesn't have to be URI reference according to the syntax in RFC.